lua/telescope-packer/init.lua

```diff
@@ -0,0 +1 @@
+return require("telescope._extensions.packer.main")
```

Comment:
Not doing this. We use the telescope interface only. This plugin is part of the org, so